NEWS.md

```diff
@@ -1,10 +1,14 @@
-# hint 1.49.1
+# hint 1.51.0
```

Comment:
You've designated this a Major change in the PR - which I guess makes sense, as it will very much not be backwards compatible. So should this be `2.0.0`?

Reply:
..at leas not backwards compatible for people hoping to reimport older datasets

Reply:
My mistake. It should be minor. I'm don't think it's backwards incompatible (or that semantic versioning really makes sense for end-user software!).

Reply:
> ..at leas not backwards compatible for people hoping to reimport older datasets

Quite. But that's implicit in this change being more broadly approved by the relevant parties. To be discussed on Friday.